src/scitokens_internal.cpp

```diff
@@ -532,6 +532,8 @@ std::string rs256_from_coords(const std::string &e_str,
         throw UnsupportedKeyException("Failed to serialize RSA public key");
     }
 #endif
+    e_bignum.release();
+    n_bignum.release();
```

Comment:
It sure is counter intuitive that adding back releases will fix a `free()` error, or for that matter deleting them in the original PR fixed a memory leak.  One would think the opposite, that calling `release()` could fix a memory leak and removing it could fix a double-free error.

Reply:
Since those variables are `unique_ptr`, `release()` in this context actually means for the `unique_ptr` to stop managing the pointer.  So, effectively, it's leaking memory.  But really, it must be passing it over to OpenSSL library somewhere.

Sometimes I am envious of memory safe languages like go, or the ownership mechanics of rust.